app/domain/queries/find_content.rb

```diff
@@ -61,4 +69,20 @@ def slice_editions
     editions = editions.where('document_type = ?', document_type) if document_type
     editions
   end
+
+  def model_name
+    return Aggregations::SearchLastThirtyDays if date_range == 'last-30-days'
+    return Aggregations::SearchLastMonth if date_range == 'last-month'
+    return Aggregations::SearchLastThreeMonths if date_range == 'last-3-months'
+    return Aggregations::SearchLastSixMonths if date_range == 'last-6-months'
+    Aggregations::SearchLastTwelveMonths
```

Comment:
I would not default to a particular view. It is better to fail than silently returning defaults

Reply:
👍 see above